static/js/common/upload-addon.js

```diff
@@ -344,7 +344,7 @@
             updateListedStatus();
 
             $('#id_is_manual_review').bind('change', function() {
-                $('.addon-upload-dependant').attr('disabled', !($(this).is(':checked')));
+                $('.addon-upload-dependant').prop('disabled', !($(this).is(':checked')));
```

Comment:
Yes probably but following the rule we made yesterday about not trying to fix cruft I'm going to err on the side of not making clean-up changes right now.